### PR TITLE
Move colour codes to common file

### PR DESCRIPTION
**Pull request by neomatrix369**
Extracted commonly used colour codes from multiple script files and made it centrally accessible to all other scripts

DO NOT MERGE THIS PR TILL #58 IS MERGED

### CODE REVIEW

**Comment by neomatrix369** [comment]
https://travis-ci.org/AdoptOpenJDK/openjdk-build/builds/221794557 failed due to rake failing

**Comment by karianna** [comment]
@neomatrix369 Conflicts.

**Comment by tellison** [comment]
This now fails the lint checks.  Take a look at the details on the Travis CI status.

**Comment by neomatrix369** [comment]
@karianna I was waiting for @tellison's pr - resolving conflicts asap

**Comment by neomatrix369** [comment]
@tellison Shellcheck isn't playing well with our idea of extracting the common colours into a separate file, the error/warnings are related to it not being able to see those variables when bash interpreter does see them and when the scripts are run we see the effects - any thoughts?

File: colour-codes.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
```
Comment (tellison) [inline colour-codes.sh]: Should add `# shellcheck disable=SC2034` to tell the lint checker to ignore unused variables errors, as this file is intended to be included in other scripts.

Comment (neomatrix369) [inline colour-codes.sh]: Thanks @tellison i'll do that, just learnt about it now, so I see why you had it across your changes :+1:

**Comment by neomatrix369** [comment]
In the green now @karianna @tellison

**Pull request merged**